src/plugins/blocksmith/components/facebook-post-block-placeholder/index.jsx

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
 import GenericBlockPlaceholder from '@triniti/cms/plugins/blocksmith/components/generic-block-placeholder';
+import Message from '@gdbots/pbj/Message';
+import PropTypes from 'prop-types';
 
-const config = {
-  iconGroup: {
-    icons: {
-      primary: {
-        imgSrc: 'pencil',
-      },
-      secondary: {
-        imgSrc: 'facebook',
+const FacebookPostBlockPlaceholder = (props) => {
+  const { block } = props;
+  const getUserName = block.getData().get('canvasBlock').get('href').split('/')[3];
+  const fromUserName = getUserName.indexOf('photo.php') === -1 ? `from ${getUserName}` : '';
```

Comment:
@tbone1020 there is simply not a reliable field for a facebook post block to show the username, it is what it is, we should not update the placeholder for this block

Reply:
resolved